<launch>
    <arg name="veh"  doc="Name of vehicle. ex: megaman"/>
    <arg name="config" default="baseline" doc="Specify a config."/>
    <arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman." />
    <!-- <arg name="testing" default="false" doc="using for test purposes." /> -->

    <arg name="pkg_name" value="veh_coordinator"/>
    <arg name="node_name" default="fake_coordinator_node"/>

    <group ns="$(arg veh)">        
        <!-- run local -->
        <node  name="$(arg node_name)" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen" required="true">
        </node>
    </group>

    <!-- Publications -->

    <!-- Subscriptions -->

    
</launch>
